Formula/sshguard.rb

```diff
@@ -16,12 +16,22 @@ class Sshguard < Formula
   depends_on "autoconf" => :build
```

Comment:
Probably not. I didn't submit this formula, I'm just modifying it to work. I'll remove those dependencies.